core/client/fs/src/main/java/alluxio/client/block/stream/DataReader.java

```diff
@@ -22,14 +22,21 @@
 public interface DataReader extends Closeable {
 
   /**
-   * The data reader type.
+   * The data reader type represents different ways to read data.
    */
   enum DataReaderType {
-    BLOCK_WORKER,
-    GRPC,
+    BLOCK_WORKER, // Read from a worker in the same process
```

Comment:
nit: If this only applies to FUSE, maybe mention this in the comment? People may not understand when you have a worker in the same process.